advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    dimensions, units, origins, rank = func.define_dimensionsUnitsOrigins(nx_effective, ny_effective, nz_effective, nhalo, mpi_comm)
</pre><p>why define effective variables if they are in implementation detail of define_dimensionsUnitsOrigin?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>